samples/smart/README.md

```diff
@@ -1,33 +1,60 @@
-# Azure SMART on FHIR ONC (g)(10) Sample
+# Azure ONC (g)(10) & SMART on FHIR Sample
 
-Sample for ONC (g)(10) / SMART on FHIR work. Still under active development.
+This sample demonstrates how Azure Health Data Services can be used to pass the Inferno test for ONC (g)(10) compliance, using Azure Active Directory as the identity provider. While the FHIR Server is the core of this sample, some custom code and routing is required to fully meet the requirements. This sample is therefore *not* using only the FHIR Server but other Azure Services to pass the Inferno tests.
 
-## Prerequisites
+## Components 
 
-This samples uses the Azure Developer CLI. Please install this via [the instructions here](https://learn.microsoft.com/azure/developer/azure-developer-cli/install-azd?tabs=baremetal%2Cwindows).
+The following components are deployed with this sample. For more details of how the pieces work together, check out [the technical guide](./docs/technical-guide.md).
 
-Open a terminal to the same directory as this README.
+- FHIR Service
+- Azure API Management
+- Azure Function
+- Azure Storage
```

Comment:
FHIR service and Azure Storage are called out as components that will be deployed with this sample. We will like to provide sample for deployed FHIR service with Storage. So should they be called out as pre-requisites.

The line should be updated to state : This sample will guide on steps to deploy components Azure API Management and Azure Function.